firestore/testapp/README.md

```diff
@@ -37,6 +37,7 @@ Building and Running the testapp
         For the latest version of the C++ SDK, you can find the associated pod versions on the
         [`Add Firebase to your project` page](https://firebase.google.com/docs/cpp/setup?platform=ios#libraries-ios).
         For instance: if you're using SDK version 8.2.0, use the following in the Podfile:
+        (but note that pod versions may not always match the C++ SDK version)
```

Comment:
Ultranit: the colon should be moved to the end of the sentence (i.e., `s/in the Podfile:/in the Podfile/` and `s/version)/version):`).